src/http_server.cc

```diff
@@ -1991,12 +1995,40 @@ HTTPAPIServer::HandleTrace(evhtp_request_t* req, const std::string& model_name)
       req, trace_response.AddString("trace_rate", std::to_string(rate)));
   RETURN_AND_RESPOND_IF_ERR(
       req, trace_response.AddString("trace_count", std::to_string(count)));
+  if (trace_mode == TRACE_MODE_TRITON) {
+    RETURN_AND_RESPOND_IF_ERR(
+        req, trace_response.AddString(
+                 "log_frequency", std::to_string(log_frequency)));
+    RETURN_AND_RESPOND_IF_ERR(
+        req, trace_response.AddString("trace_file", filepath));
+  }
+  std::string trace_mode_lower =
+      trace_manager_->InferenceTraceModeString(trace_mode);
```

Comment:
@oandreeva-nv @indrajit96 can we just change `InferenceTraceModeString` to return the lowercase variants instead of transforming it?